en/simulation/README.md

```diff
@@ -129,6 +129,25 @@ make px4_sitl jmavsim
 <span></span>
 > **Note** To avoid PX4 detecting data link timeouts, increase the value of param [COM_DL_LOSS_T](../advanced/parameter_reference.md#COM_DL_LOSS_T) proportional to the simulation rate. For example, if `COM_DL_LOSS_T` is 10 in realtime, at 10x simulation rate increase to 100.
 
+### Lockstep Simulation
```

Comment:
Should this be a lower level heading? Ie it is part of "Run Simulation Faster than Realtime"

Reply:
Yes, can we have `####`?

Reply:
Yes, we can have up to 5 or 6, but they get progressively less easy to differentiate. I actually ended up leaving this at the same level, but I put disabling lockstep underneath it.

Reply:
Ok fine.